test/controllers/unattended_controller_test.rb

```diff
@@ -13,19 +13,19 @@ class UnattendedControllerTest < ActionController::TestCase
   context "redhat" do
     setup do
       ptable = FactoryBot.create(:ptable, :name => 'default',
-                                  :operatingsystem_ids => [operatingsystems(:redhat).id])
+                                 :operatingsystem_ids => [operatingsystems(:redhat).id])
```

Comment:
why don't / can't you switch to current hash syntax if you are touching this anyway? (question applies to all other places as well)

Reply:
I usually do this, the old syntax gives me the creeps. Actually, I want to revert these changes to make the diff easier to read as this is clearly unrelated.

Reply:
IIRC there's a rubocop for this. It's probably good to standardize on this in a separate issue.

Reply:
Problem I see with making a change like this everywhere in a separate issue is it will cause allot of merge conflicts for existing issues. Hence I think the only plausible way without havoc is to change to the new syntax where/when ever changes are made.

Reply:
Agreed @leewaa but we tend to slowly enable various cops one by one in different patches. Our codebase is huge and there are many to go: https://github.com/theforeman/foreman/blob/develop/.rubocop_todo.yml

Actually not that many anymore, great job everyone!

Reply:
@leewaa agreed one big PR is disrupting. I mostly meant by different issue that it shouldn't be part of this PR :)

Reply:
Yes very difficult to do this with such a large code base.

Reply:
Anyway, can you revert those if you are not touching the lines?